### DISCUSSION TITLE
Ionospheric correction for Sentinel-1

### DISCUSSION THREAD

**Discussion by fdelgadodelapuente** [isce-framework/isce2#691] [Q&A]
Dear ISCE developers,

I have a question about the ionospheric signals that are corrected for TOPS interferograms. I've noticed that in several cases in the Central Andes of northern Chile and Argentina the correction can only estimate a long wavelength ramp, but can not mitigate the jumps in the burst overlap regions that can be easily identified in the combined_IW?.int files. Therefore why does it happen? Is it possible to "force" the algorithm to correct the jumps between the different bursts instead of the long wavelength ramps?

Thanks,
Francisco

**Comment by CunrenLiang** [comment]
Francisco,

This is the option you need to turn on
https://github.com/isce-framework/isce2/blob/main/examples/input_files/topsApp.xml#L187

However, it cannot do a perfect correction for the burst phase jumps.

Cunren

**Comment by fdelgadodelapuente** [comment]
Thank you Cunren. I know it's not perfect, but I'll give it a try. 

Francisco
